### ISSUE TITLE
Unhandled page fault when running Epic Games Store

### ISSUE THREAD

**Issue by happysmash27** [lutris/wine#63]
After some updates, when I try to launch the Epic Games Store using the Lutris-5.7.8 runner, the window appears for a bit, loading, then closes, with the following error: 

```
wine: Unhandled page fault on read access to 0000000000000000 at address 0000000009C0587B (thread 0034), starting debugger...
Caught signal 15
--terminated processes--
```

This error does not occur with Wine-Staging 5.17, but I get much lower framerates on Wine-Staging. 

I have to kill all Wine-related processes manually to stop the Epic Games Store completely. I'm not sure where the debugger the log is referring to is; it appears to be inaccessible.

**Comment by happysmash27** [comment]
I realised I had manually selected a Vulkan ICD loader, and changed it back to auto, as the runner appeared to work when set to auto with WATCH_DOGS 2 (although with the same graphics glitch where everything is green I've had ever since trying to run it), but this still yields the exact same error.

**Comment by happysmash27** [comment]
This seems to be able to be resolved by switching from 5.7 to 6.0-rc1. Although it's still a little slower, it runs pretty close to how it used to on the low-end.
